carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/Configuration.java

```diff
@@ -545,4 +544,7 @@ public static void setMobileApp(String mobileApp) {
     public static Object getCapability(String name) {
         return R.CONFIG.get("capabilities." + name);
     }
+    public static String getHubMode() {
+        return R.CAPABILITIES.get("capabilities.provider");
```

Comment:
1. value should be read from CONFIG.
2. inside [config.properties](https://github.com/qaprosoft/carina/blob/master/carina-core/src/main/resources/config.properties) 
```
hub_mode=selenium
->
capabilities.provider=selenium
```
3. [HubType](https://github.com/qaprosoft/carina/blob/c251cd07bde6a3de546547a459be01d063a9d3be/carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/core/factory/DriverFactory.java#L52) enum declaration should be removed.